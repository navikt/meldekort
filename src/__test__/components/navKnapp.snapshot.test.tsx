import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper, {
  setLocalesBeforeAll,
} from '../testSetup/providerWrapper';
import NavKnapp, { KnappTyper } from '../../app/components/knapp/navKnapp';

setLocalesBeforeAll();

it('NavKnapp', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <NavKnapp
        type={KnappTyper.HOVED}
        nestePath={'/test'}
        tekstid={'naviger.neste'}
      />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
