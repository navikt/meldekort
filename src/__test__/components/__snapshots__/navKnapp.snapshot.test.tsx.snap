// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavKnapp 1`] = `
<button
  className="knapp knapp--hoved"
  disabled={false}
  onClick={[Function]}
  type="submit"
>
  <span>
    Neste
  </span>
</button>
`;
